pcsx2/gui/Panels/GameFixesPanel.cpp

```diff
@@ -93,7 +93,7 @@ Panels::GameFixesPanel::GameFixesPanel( wxWindow* parent )
 			wxEmptyString
 		},
 		{
-			_("Ignore Bus Direction on Path3 Transfer - Used for Hotwheels"),
+			_("Enable the GIF FIFO (slower but needed for Hotwheels, Wallace & Gromit, DJ Hero)"),
```

Comment:
I just noticed it's also in the game fix above it:
"Delay VIF1 Stalls (VIF1 FIFO) - For SOCOM 2 HUD `&` Spy Hunter loading hang."